src/app/courses/courses.component.ts

```diff
@@ -1,28 +1,31 @@
-import { Component, OnInit } from '@angular/core';
-import { Courses, CoursesService } from '../shared/services/courses.service';
-import { ActivatedRoute, Router, NavigationEnd } from '@angular/router';
-import { SubscriptionsManager } from '../shared/base/subscriptions-manager';
-
+import { Component, OnInit } from "@angular/core";
+import { Courses, CoursesService } from "../shared/services/courses.service";
+import { ActivatedRoute, Router, NavigationEnd } from "@angular/router";
+import { SubscriptionsManager } from "../shared/base/subscriptions-manager";
 
 @Component({
-  selector: 'app-courses',
-  templateUrl: './courses.component.html',
-  styleUrls: ['./courses.component.scss']
+  selector: "app-courses",
+  templateUrl: "./courses.component.html",
+  styleUrls: ["./courses.component.scss"]
 })
 export class CoursesComponent extends SubscriptionsManager implements OnInit {
   coursesList: Courses[];
   courseSelected: boolean;
-  constructor(private coursesService: CoursesService, private router: Router, private route: ActivatedRoute) {
+  constructor(
+    private coursesService: CoursesService,
+    private router: Router,
+    private route: ActivatedRoute
+  ) {
     super();
   }
 
-
   ngOnInit() {
     this.coursesList = this.coursesService.getAll();
 
-    this.subs = this.router.events.subscribe((event) => {
+    this.subs = this.router.events.subscribe(event => {
       if (event instanceof NavigationEnd) {
-        this.courseSelected = !!(this.route.firstChild.snapshot.params.id as string);
+        this.courseSelected = !!(this.route.firstChild.snapshot.params
+          .id as string);
```

Comment:
@Yuriy-27 why you moved `.id as...` to the new line, it was looking ok I believe